source/docs/partnering/experts/spec.yml

```diff
@@ -15,103 +14,198 @@ securityDefinitions:
 security:
   - api_key: []
 paths:
-  /experts:
+  /api/v1/experts:
     get:
       summary: Returns a list of national laboratory experts
       parameters:
-      - in: query
-        name: lab_alias
-        type: string
-        enum:
-          - ames
-          - anl
-          - bnl
-          - fnal
-          - inl
-          - kcnsc
-          - lanl
-          - lbnl
-          - llnl
-          - netl
-          - nrel
-          - ornl
-          - pantex
-          - pnnl
-          - pppl
-          - slac
-          - snl
-          - srnl
-          - tjnaf
-          - y12
-        required: false
-        description: National laboratory alias, when not specified all experts are returnd from all labs
+        - in: path
+          name: version
+          type: integer
+          required: true
+          description: API version
+        - in: query
+          name: limit
+          type: integer
+          required: false
+          description: Number of results to return, default is 25, max is 200; to receive more than <i>limit</i> number of results, increment page param in subsequent request.
+        - in: query
+          name: page
+          type: integer
+          required: false
+          description: Page number of results, default is 0; see limit
       responses:
         200:
           description: Successful request
           schema:
             $ref: "#/definitions/experts_response"
-  /experts/{expert_uuid}:
+          examples:
+            Valid:
+              metadata: {"version": 1}
+              inputs: {}
+              errors: {}
+              resultset: {
+                "total": 141,
+                "count": 25,
+                "number": 0,
+                "pages": 6,
+                "result": [
+                {
+                "uuid": "a86f59b3-f23c-4c4f-94bb-386cdfea7ce5",
+                "first_name": "joel w.",
+                "last_name": "ager",
+                "avatar": "https://www.labpartnering.org/files/experts/a86f59b3-f23c-4c4f-94bb-386cdfea7ce5",
+                "links": {"self": "https://www.labpartnering.org/api/v1/experts/a86f59b3-f23c-4c4f-94bb-386cdfea7ce5"},
```

Comment:
Are these example URLs correct? These appear to be returning URLs to where the underlying API lives, which you may not want to be publicly accessible if you want to ensure people use API keys and are rate limited: https://github.com/18F/api.data.gov/wiki/User-Manual:-Agencies#securing-your-api-backend

Reply:
These will also be updated by our push to prod (hopefully) today.

Reply:
The URLs in question have been live since yesterday.  It would be good if we could merge this pull request so the documentation is available.

Reply:
I might be misunderstanding you here, but I still think these new URLs in the docs might be incorrect. The URLs referenced in the docs need to be the public URLs that external users access via developer.nrel.gov (which may rewrite the path before passing it along to your underlying API backend). These updated Swagger docs point to the API existing at https://developer.nrel.gov/api/v1/experts, but those API requests will fail because nothing is there. Instead, I believe the URL paths have actually remained the same on developer.nrel.gov's end (at least for the 1 experts API), since the API still seems to live at: https://developer.nrel.gov/api/lps/v1/experts?api_key=DEMO_KEY (note the `/api/lps/v1` prefix, instead of `/api/v1` prefix).

The URLs in these new examples also don't seem to match what I get when I hit the API: https://developer.nrel.gov/api/lps/v1/experts?api_key=DEMO_KEY returns `"self"` links pointing to `https://developer.nrel.gov/api/lps/v1/*`, which I think is correct. However, these updated Swagger examples are indicating the API returns those URLs as `https://www.labpartnering.org/api/v1/*` which doesn't seem to be correct (since those URLs aren't publicly accessible).

If you need to adjust the URL routing for any of this, you can review that in the api.data.gov admin: https://api.data.gov/admin/#/apis

But let me know if this isn't making sense, or I'm missing something here.

Reply:
Ah, thanks, I was looking at the paths and overlooked the domains.  I just pushed fixes for those and published the production changes in API Umbrella.

Reply:
Cool, these updates to the example URLs look good. However, I think there are still issues with the changes to `basePath` and `paths`. If I try to use these Swagger docs after these changes, it's trying to make API requests to https://developer.nrel.gov/api/v1/experts/api/v1/experts, which isn't the correct public URL.

Reply:
Right, I did still have path errors, and some type errors for the version param too.  I just pushed corrections such that all of these have basePath /api/lps and the item paths are relative to that, with the version properly dynamic.  I could have set basePath to / and made all the sub-paths complete, but that was looking cluttered.  I'm open to suggestion, but the try-it-out mechanism should work now.  Thanks for your help!

Reply:
I just pushed a change such that all basePaths are now /, and the paths are complete.  Should be less confusing, and it's not that cluttered.

Reply:
Reminder, I believe this is straightened out now.